sw/airborne/modules/px4_gimbal/px4_gimbal.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) Kevin van Hecke
+ *
+ * This file is part of paparazzi
+ *
+ */
+/**
+ * @file "modules/px4_gimbal/px4_gimbal.c"
+ * @author Kevin van Hecke
+ * Control gimbal camera axis through px4 from rc
+ */
+
+#include "modules/px4_gimbal/px4_gimbal.h"
+#include "subsystems/radio_control.h"
+
+#include "subsystems/datalink/downlink.h"
+#include "generated/airframe.h" // AC_ID is required
+#include "subsystems/datalink/datalink.h"
+#include "subsystems/datalink/telemetry.h"
+#include "subsystems/actuators.h"
+
+#include BOARD_CONFIG
+#include DEFAULT_ACTUATORS
+
+#ifndef PX4_GIMBAL_PWM_CHAN
+#define PX4_GIMBAL_PWM_CHAN 0
+#endif
+#ifndef PX4_GIMBAL_RC_CHAN
+#define PX4_GIMBAL_RC_CHAN RADIO_AUX2
+#endif
+
+#ifndef PX4_GIMBAL_PWM_MIN
+#define PX4_GIMBAL_PWM_MIN 900
+#endif
+
+#ifndef PX4_GIMBAL_PWM_MAX
+#define PX4_GIMBAL_PWM_MAX 1500
+#endif
+
+
+void px4_gimbal_init() {
+#ifdef INTER_MCU_AP
+   actuators_init();
+#endif
+}
+
+ void px4_set_gimbal_angle_event() {
+   //reroute gimbal rc input from fbw to pwm output on ap
+
+   float value =radio_control.values[PX4_GIMBAL_RC_CHAN];
+
+   float min = 900;
```

Comment:
I guess you want to use `PX4_GIMBAL_PWM_MIN` here... also doesn't seem to really be a float...